File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Data/GeneralClass.vb
Imports System.Security.Cryptography
Public Class GeneralClass
    Public Shared Property IdUsuario As Integer

    Public Shared Function SHA1(ByVal text As String) As String
        'Dim data = Encoding.ASCII.GetBytes(text)
        'Dim data = Encoding.ASCII.GetBytes(text)
        'Dim hashData = New SHA1Managed().ComputeHash(data)
        Dim hash = String.Empty

        'For Each b In hashData
        'hash += b.ToString("X2")
        'Next

        Return hash.ToLower()
    End Function

End Class

File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Data/GeneralClass.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Security.Cryptography;

namespace DDS_Restaurant_Solution.Data
{
    public class GeneralClass
    {
        public static int IdUsuario { get; set; }
        public static string SHA1(string text)
        {
            var data = Encoding.ASCII.GetBytes(text);
            var hashData = new SHA1Managed().ComputeHash(data);
            var hash = string.Empty;

            foreach (var b in hashData)
            {
                hash += b.ToString("X2");
            }

            return hash.ToLower();
        }
    }
}
